### PR TITLE
Fix/console issue2

### PR DESCRIPTION
**Pull request by KirillSerg**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "react-bootstrap-icons": "^1.8.4",
     "react-bootstrap-typeahead": "^6.0.0-rc.3",
     "react-dom": "^18.2.0",
-    "react-helmet": "^6.1.0",
+    "react-helmet-async": "^1.3.0",
```
Comment (alexandrtovmach) [inline package.json:52]: why?

File: src/containers/InfiniteList.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const InfiniteList = <T extends object>({
       loadMore={showMore}
       hasMore={data.hasNextPage}
       loader={
-        <div className="m-2 d-flex justify-content-center">
+        <div key={0} className="m-2 d-flex justify-content-center">
```
Comment (alexandrtovmach) [inline src/containers/InfiniteList.tsx]:
```suggestion
        <div key="infinite_scroll_spinner" className="m-2 d-flex justify-content-center">
```

File: src/containers/Layout.tsx [hunk 3]
```diff
@@ -17,19 +17,21 @@ interface LayoutProps {
...
+          <ErrorsToaster />
+          <div className="row">
+            <LeftPanel />
+            <main className="col-xl-6 col-lg-6 col-md-11 col-sm-11 col-12">
+              {children}
+            </main>
+            <RightPanel />
+          </div>
+        </GeneralContainer>
+      </HelmetProvider>
```
Comment (alexandrtovmach) [inline src/containers/Layout.tsx:34]: is it not possible to resolve error with normal react-helmet? could you add reference to github issue/stackoverflow where you found this solution

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (KirillSerg) [reply package.json:52]: https://github.com/nfl/react-helmet/issues/548

File: src/containers/Layout.tsx (hunk 3 above)
Comment (KirillSerg) [reply src/containers/Layout.tsx:34]: https://github.com/nfl/react-helmet/issues/548

File: package.json (hunk 1 above)
Comment (alexandrtovmach) [reply package.json:52]: makes sense then

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**